readme.md

```diff
@@ -168,72 +384,67 @@ var Hello = React.createClass({
   }
 });
 
-React.render(
+ReactDOM.render(
   <Hello name="Tony" age="21" />,
-  document.getElementById( "container" )
+  document.getElementById( "app" )
 )
 ```
 
-## Exercise: A Blog Post (15 / 45)
+> **NOTE:** The return statement in `render` can only return one DOM element. You can, however, place multiple elements within a parent DOM element, like we do in the previous example with `<div>`.
+
+## Exercise: A Blog Post
 
 Let's have some practice creating a React component for scratch. How about a blog post?
-* Define a Post constructor that has the following properties:
-  1. title
-  2. author
-  3. body
-  4. comments
+* Define a Post constructor in `index.js` that has the below properties. Then use it to generate an instance of a Post with the content of your choosing.
+  1. `title`
+  2. `author`
+  3. `body`
+  4. `comments` (array of strings)
 * Render these properties using a Post component.
-* The HTML/CSS composition of your Post is up to you.
-
-### Solution
+* The HTML (or more accurately, JSX) composition of your Post is up to you.
 
-[https://gist.github.com/amaseda/efeb9d6ee266f8fa3178](https://gist.github.com/amaseda/efeb9d6ee266f8fa3178)
+#### [Solution](https://gist.github.com/amaseda/efeb9d6ee266f8fa3178)
 
-## Nested Components (5 / 50)
+## Nested Components
 
-What problems did you encounter when trying to add multiple comments to your Post?
+**Q:** What problems did you encounter when trying to add multiple comments to your Post?
 * It would be a pain to have to explicitly define every comment inside of `<PostView />`, especially if each comment itself had multiple properties.
-* The solution? **Nested components**.
 
-We can nest Comment components within a PostView component
+We can nest Comment components within a PostView component.
 * We create these comments the same way we did with posts: `.createClass` and `.render`
 * Then we can reference a comment using `<Comment />` inside of PostView's render method.
 
-## Exercise: Add Nested Comments To Blog (10 / 60)
+## Exercise: Add Nested Comments To Blog
 
-1. Create a `CommentView` component in the same way we did for `PostView`.
-* Your `CommentView` render method should render a `commentBody` property.
+1. Create a `CommentView` component in the same way we did for `PostView`. Its `render` method should render a `commentBody` property.
```

Comment:
thoughts on removing `View` from component definitions?  Not a big deal, but just won't be following that convention for any other code example